src/mail/editor/MailEditor.ts

```diff
@@ -853,11 +832,11 @@ export class MailEditor implements Component<MailEditorAttrs> {
  * Creates a new Dialog with a MailEditor inside.
  * @param model
  * @param blockExternalContent
- * @param isReplyingFromEmailView Inform if it's a reply to an email
+ * @param alwaysBlockExternalContent
  * @returns {Dialog}
  * @private
  */
-async function createMailEditorDialog(model: SendMailModel, blockExternalContent: boolean = false, isReplyingFromEmailView: boolean = false): Promise<Dialog> {
+async function createMailEditorDialog(model: SendMailModel, blockExternalContent = false, alwaysBlockExternalContent = false): Promise<Dialog> {
```

Comment:
I thought `alwaysBlockExternalContent` would be `true` in most cases but you would like to show the banner unless the rule is defined, right?

Reply:
Yes, if the user defined "Always Block" I don't want to bother they again asking to load the images